index.js

```diff
@@ -176,25 +176,29 @@ function redisStore(args) {
    * @param {Function} cb - A callback that returns a potential error and the response
    */
   self.get = function(key, options, cb) {
-    if (typeof options === 'function') {
-      cb = options;
-      options = {};
-    }
-    options = options || {};
-    options.parse = true;
+    return new Promise(function(resolve, reject) {
```

Comment:
@lukechilds Can we pick Promise from promiseDependency (here)[https://github.com/BryanDonovan/node-cache-manager#example-using-promises] so that we honour the users promise implementation choice ?